src/demo/models/contracts/index.js

```diff
@@ -268,10 +269,8 @@ const buildFormLayout = viewName => ({ tab, section, field }) => instance => [
   tab({ name: 'general', columns: 2 }, // Best look with N = 2, 3, 4 (default is 1)
     field({ name: 'contractId', readOnly: viewName !== VIEW_CREATE }),
     field({ name: 'description' }),
-    // field({ name: 'translations', render: { Component: TranslatableTextEditor }}),
     field({ name: 'statusId', render: { Component: StatusField, valueProp: { type: 'number' } } }),
-    // field({ name: 'parentContract', render: { Component: ContractReferenceSearch }}),
-    // field({ name: 'currencyId', render: { Component: CurrencyField }}),
+    field({ name: 'parentContract', render: { Component: ReferenceSearch, valueProp: { type: 'string' } } }),
```

Comment:
remove valueProp because it has default values

Reply:
Done